### PR TITLE
Plop: class / function choices for generation

### PR DESCRIPTION
**Pull request by romansorin**
Implements #45 functionality.

### CODE REVIEW

File: config/plop/component/function/component.styled.js.plop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import * as S from './styles'
+
+const {{pascalCase componentName}} = () => {
+   return <S.{{pascalCase componentName}}Component> 🚀 {{componentName}} </S.{{pascalCase componentName}}Component>
+}
+
+export default {{pascalCase componentName}}
```
Comment (romansorin) [inline config/plop/component/function/component.styled.js.plop]:
```suggestion
export default {{pascalCase componentName}}

```
Newline

File: config/plop/component/component.generator.js [hunk 2]
```diff
@@ -13,34 +13,84 @@ module.exports = {
...
-      }, {
-        type: 'addComponentToIndex'
-      })
+        path: `${
+          paths.components
+        }/{{properCase componentName}}/{{properCase componentName}}.js`,
+        templateFile: './config/plop/component/function/component.js.plop'
+      },
+      styledClass: [
+        {
```
Comment (RyanWarner) [inline config/plop/component/component.generator.js]: I'd like to keep the object array whitespace the same. I know it's personal preference, but I don't think standardjs is opinionated on this.

Comment (romansorin) [reply config/plop/component/component.generator.js]: Sure, is there some way to change this in the Standard formatter extension? This is the default behavior:

Comment (RyanWarner) [reply config/plop/component/component.generator.js]: Hmmm, I don't use this extension. I just rely on ESLint. I wonder if they have slightly different rulesets or what. Anyways, I'll look into that later as it's not super important.

**Pull request merged**